pilot/pkg/networking/core/v1alpha3/httproute.go

```diff
@@ -375,9 +377,11 @@ func BuildSidecarOutboundVirtualHosts(node *model.Proxy, push *model.PushContext
 	// Get list of virtual services bound to the mesh gateway
 	virtualHostWrappers := istio_route.BuildSidecarVirtualHostWrapper(routeCache, node, push, servicesByName, virtualServices, listenerPort)
 
-	resource, exist := xdsCache.Get(routeCache)
-	if exist && !features.EnableUnsafeAssertions {
-		return nil, resource, routeCache
+	if features.EnableRDSCaching {
```

Comment:
for get we donot need this

Reply:
why is that? for cds we do respect it for get as well
https://github.com/istio/istio/blob/1.14.3/pilot/pkg/networking/core/v1alpha3/cluster_builder.go#L1271